index.html

```diff
@@ -376,6 +377,18 @@ <h2>
           The Gamepad's controls have been mapped to the <a href=
           "#remapping">Standard Gamepad layout</a>.
         </dd>
+        <dt>
+          <dfn>xr-standard</dfn>
+        </dt>
+        <dd>
+          The Gamepad's controls have been mapped to the <a href=
+          "https://immersive-web.github.io/webxr-gamepads-module/#xr-standard-gamepad-mapping">
+          XR Standard Gamepad Layout</a>. This mapping is reserved for use by
```

Comment:
@toji, in webxr-gamepads-module, would you mind `<dfn export>`ing XR Standard Gamepad Layout? Then we can link to it like so:  

```suggestion
          The Gamepad's controls have been mapped to the [=XR Standard
          Gamepad Layout=]. This mapping is reserved for use by
```

Reply:
Added the export to https://github.com/immersive-web/webxr-gamepads-module/pull/36, but we were waiting to land that until this change is merged so that we don't lose the enum definition, even temporarily.